deploy.sh

```diff
@@ -23,5 +23,5 @@ sam deploy --stack-name "$stack_name" \
    Environment=dev \
    AuditEventNamePrefix=/common-cri-parameters/KbvAuditEventNamePrefix \
    CriIdentifier=/common-cri-parameters/KbvCriIdentifier \
-   CommonStackName=kbv-common-cri-api-local \
```

Comment:
I think this needs to be left as it was.

Reply:
this change has been reverted